maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -151,6 +155,14 @@ const LayerControlPanel = memo(({ maplibreRef, setLayers, layers }: Props) => {
     layerVisibility.set(layer.id, layer.visibility === LAYER_VISIBILITY.VISIBLE);
   });
 
+  const onDragEnd = ({ source, destination }) => {
+    if (source && destination) {
+      const items = euiDragDropReorder(layers, source.index, destination.index);
```

Comment:
How about use newOrderLayers instead of items?

Reply:
updated